mailbox/store/src/main/java/org/apache/james/mailbox/store/mail/MailboxMapper.java

```diff
@@ -38,11 +38,16 @@
  *
  */
 public interface MailboxMapper extends Mapper {
-    
+
+    /**
+     * Create the given {@link Mailbox} to the underlying storage
+     */
+    MailboxId create(Mailbox mailbox) throws MailboxException;
```

Comment:
I bet a next step will be `Mailbox create(MailboxPath, UidValidity)`

Reply:
Other PR :)

Reply:
#3128